doc/contribute/guidelines.rst

```diff
@@ -643,53 +649,141 @@ workflow here:
    Additional information about the CI system can be found in
    `Continuous Integration`_.
 
+.. _commit-guidelines:
+
+Commit Message Guidelines
+*************************
+
+Changes are submitted as Git commits. Each commit has a *commit
+message* describing the change. Acceptable commit messages look like
+this:
+
+.. code-block:: none
 
-Commit Guidelines
-*****************
+   [area]: [summary of change]
 
-Changes are submitted as Git commits. Each commit message must contain:
+   [Commit message body (must be non-empty)]
+
+   Signed-off-by: [Your Full Name] <[your.email@address]>
+
+You need to change text in square brackets (``[like this]``) above to
+fit your commit.
+
+Examples and more details follow.
+
+Example
+=======
 
-* A short and descriptive subject line that is less than 72 characters,
-  followed by a blank line. The subject line must include a prefix that
-  identifies the subsystem being changed, followed by a colon, and a short
-  title, for example:  ``doc: update wiki references to new site``.
-  (If you're updating an existing file, you can use
-  ``git log <filename>`` to see what developers used as the prefix for
-  previous patches of this file.)
+Here is an example of a good commit message.
 
-* A change description with your logic or reasoning for the changes, followed
-  by a blank line. (Every single line has to be less than 75 characters.)
+.. code-block::
```

Comment:
Use of  "code-block" here is causing keywords such as "for" , "and", etc  to get highlighted in the html output.  Is there another tag that can be used to get fixed width formatting?

Reply:
> Use of "code-block" here is causing keywords such as "for" , "and", etc to get highlighted in the html output. Is there another tag that can be used to get fixed width formatting?

This should have been `code-block:: none`. Thanks for catching that. Fixed.